examples/prepy310/example.py

```diff
@@ -3,7 +3,7 @@
 
 from example_utils.example_logging import initialize_logging
 
-from momento import SimpleCacheClient
+from momento import CacheClient
```

Comment:
you can't do the examples in the same PR, right?

Reply:
Nope! My recursive `sed` call didn't know about that :P